composer.json

```diff
@@ -33,17 +33,23 @@
   },
   "require-dev": {
     "composer/composer": "1.6.* || 2.0.*@dev",
-    "composer/semver": "^1 || ^3"
+    "composer/semver": "^1 || ^3",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
+    "wearerequired/coding-standards": "^1.5"
   },
   "extra": {
     "branch-alias": {
-      "dev-master": "1.0-dev"
+      "dev-master": "0.4.x-dev"
```

Comment:
What is the reasoning here?

Reply:
There is no 1.0 yet, the next one is 0.4.